src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -41,6 +63,9 @@ static Microsoft.CodeAnalysis.Rename.DocumentRenameOptions.operator !=(Microsoft
 static Microsoft.CodeAnalysis.Rename.DocumentRenameOptions.operator ==(Microsoft.CodeAnalysis.Rename.DocumentRenameOptions left, Microsoft.CodeAnalysis.Rename.DocumentRenameOptions right) -> bool
 static Microsoft.CodeAnalysis.Rename.SymbolRenameOptions.operator !=(Microsoft.CodeAnalysis.Rename.SymbolRenameOptions left, Microsoft.CodeAnalysis.Rename.SymbolRenameOptions right) -> bool
 static Microsoft.CodeAnalysis.Rename.SymbolRenameOptions.operator ==(Microsoft.CodeAnalysis.Rename.SymbolRenameOptions left, Microsoft.CodeAnalysis.Rename.SymbolRenameOptions right) -> bool
+virtual Microsoft.CodeAnalysis.CodeRefactorings.CodeRefactoringProvider.GetFixAllProvider() -> Microsoft.CodeAnalysis.CodeRefactorings.FixAllProvider
+virtual Microsoft.CodeAnalysis.CodeRefactorings.DocumentBasedFixAllProvider.GetFixAllTitle(Microsoft.CodeAnalysis.CodeRefactorings.FixAllContext fixAllContext) -> string
+virtual Microsoft.CodeAnalysis.CodeRefactorings.FixAllProvider.GetSupportedFixAllScopes() -> System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.CodeFixes.FixAllScope>
```

Comment:
📝 It would be better if Public API Analyzers marked experimental APIs in some manner.